invenio_pages/__init__.py

```diff
@@ -1,12 +1,50 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of Invenio.
-# Copyright (C) 2015-2022 CERN.
+# Copyright (C) 2015-2023 CERN.
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Static pages module for Invenio."""
+r"""Static pages for Invenio.
+
+Invenio-pages provides means to add static pages to Invenio. 
+Once the module is installed, developers will be able to customise
+their Invenio instance by registering custom static pages in a 
+pages.yaml file.
+Define static pages
+----------
+To add static pages, the developer must create (if it doesn’t exist) a 
+pages.yaml file inside the app_data folder of the site.
+.. code-block:: python
+    <your_site>/
+    └──app_data/
+        └── pages.yaml
+Once the file is created the developer has to define in it the static 
+pages it wishes to create, using the following format for each page:
+.. code-block:: python
+    - url: <URL used to access the page> eg: /about
+      title: <Title of the page> eg: About
+      description: <Description of the content of the page (won’t appear on the page itself)> eg:About
+      template: <Html template with the content of the page> eg: about.html 
+All the templates defined in this pages.yaml file should be created 
+inside a pages folder inside the app_data folder of the site.
+.. code-block:: python
+    <your_site>/
+    └──app_data/
+            └── pages/
+                    └── about.html
+                    └── …
+Create pages
+----------
+Once the static pages have been defined, they can be created in two ways.
+- Creating all the data of the instance by running the ``invenio-cli 
+services setup`` command on the instance root. In case the data was already 
```

Comment:
This module is not aware of `invenio-cli` (or InvenioRDM in general), so we can't mention its usage here. 

Since that is the case I'm even a bit hesitant on whether these docs should be here or in InvenioRDM docs (cc @kpsherva)

Reply:
The whole module assumes being part of an RDM instance:
- `app_data` folder
- The command is actually created as part of fixtures in RDM (see more in the [epic](https://github.com/inveniosoftware/invenio-app-rdm/issues/1989) issue).

For now I would just add a note at the beginning saying "InvenioRDM" instead of "Invenio" instance. However, in the long term maybe we have to disentangle them.

Reply:
to be moved to RDM docs indeed.